recorder.js

```diff
@@ -33,6 +36,27 @@ function parse_game_stats(){
             }
         });
     });
+    if (team_game) {
+        let team_one_score = document.querySelector('[data-hook=team1-score]').textContent;
```

Comment:
I see some redundancy and symmetry in this section of the code. Can you think of a way to tighten this up so future changes are easier?

Reply:
I fiddled with
```
        let [team_one_score, team_two_score] = Array.from(
            document.querySelectorAll(
                'span[data-hook^=team][data-hook$=-score]'
            )
        ).map(
            (element) => element.textContent
        );
```
as a streamlining of this, but realized I could just grab them out of the scoreboard loop, which I think works better.